README.md

```diff
@@ -118,7 +118,12 @@ cargo install --git https://github.com/Peltoche/lsd.git --branch master
 
 ### From Binaries
 
-The [release page](https://github.com/Peltoche/lsd/releases) includes precompiled binaries for Linux and macOS.
+The [release page](https://github.com/Peltoche/lsd/releases) includes precompiled binaries for Linux and macOS, you can download the binary install lsd via:
+
+``` sh
+ tar zxvf lsd-0.17.0-x86_64-apple-darwin.tar.gz # adapt version number and architecture
+ mv lsd-0.17.0-x86_64-apple-darwin/lsd /usr/local/bin/
```

Comment:
I think it is better to keep it how it used to be.
For those who know what to do, we probably should not add anything and for those who don't know what they are doing this might only cause confusion if they are using anything other than macOS.

Reply:
I will take your `install.sh` advice, and add one here.

this is how I will do:

1. change GitHub workflow to release assets with name ``lsd-`uname -s`-`uname-m`-`libc`.tar.gz`` and dir ``lsd-`uname -s`-`uname-m`-`libc``` (remove the version)
2. with `1.` we have the static link for latest release: ``https://github.com/Peltoche/lsd/releases/download/latest/lsd-`uname -s`-`uname -m`-`libc`.tar.gz``
3. add `install.sh` to detect libc, download the latest binary and move lsd to `/usr/local/bin`

what do u think about it?

Reply:
I am not sure if changing the release asset names might cause issues with scripts that might be in place for package manager submissions. Not sure how that might work out.
I think it would be better to just handle it in the `install.sh` script.

Reply:
We have to update the download link in `install.sh` after every release if there is a version in asset's name

Reply:
I am guessing we should be able to grep it out of `https://api.github.com/repos/Peltoche/lsd/releases/latest`.

I have been using https://github.com/meain/homebrew-meain/blob/master/update for some time in one of my repos and it seem to work pretty fine.

Reply:
sounds reasonable and keeps a minimal change, let's do it this way